bundle/language_en.properties

```diff
@@ -1510,6 +1510,7 @@ angal.visit.addvisit2.btn
 angal.visit.addvisit2.btn.key                                                                          = V
 angal.visit.arrowprevious.btn                                                                          = <-Prev.
 angal.visit.arrowprevious.btn.key                                                                      = P
+angal.visit.cannotsetadateinthepastfornextvisit.msg                                                    = You can not set a date in the past for next visit.
```

Comment:
it would be better to change `You can not set a date in the past for next visit.` to be `You cannot set a date in the past for the next visit.`

Reply:
Done, thanks!